app/www/index.html

```diff
@@ -8,9 +8,170 @@
+			  <td><textarea class="textarea1 text-right" id="collaboration_3" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="contribution_3" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="attitude_3" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="goals_3" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="average_3" rows="1" cols="10" maxlength="10"></textarea></td>
+			</tr>
+			<tr>
+			  <td>Group Member 4</td>
+			  <td><textarea class="textarea1 text-right" id="participation_4" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="collaboration_4" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="contribution_4" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="attitude_4" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="goals_4" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1" id="average_4" rows="1" cols="10" maxlength="10"></textarea></td>
+			</tr>
+			<tr>
+			  <td>Group Member 5</td>
+			  <td><textarea class="textarea1 text-right" id="participation_5" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="collaboration_5" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="contribution_5" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="attitude_5" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="goals_5" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="average_5" rows="1" cols="10" maxlength="10"></textarea></td>
+			</tr>
+			<tr>
+			  <td>Group Member 6</td>
+			  <td><textarea class="textarea1 text-right" id="participation_6" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="collaboration_6" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="contribution_6" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="attitude_6" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="goals_6" rows="1" cols="10" maxlength="10"></textarea></td>
+			  <td><textarea class="textarea1 text-right" id="average_6" rows="1" cols="10" maxlength="10"></textarea></td>
+			</tr>
+			<tr class="ratings-table-colavg">
```

Comment:
This row should be centered (and gray background). Actually looks like you just have a typo in the class name:
```suggestion
			<tr class="ratings-table__colavg">
```